host.go

```diff
@@ -695,4 +698,193 @@ func (h *Host) InitCommands(c *chat.Commands) {
 			return nil
 		},
 	})
+
+	c.Add(chat.Command{
+		// TODO: find a better name for reverify
+		// TODO: default for reload
+		// TODO: add keys for a specific duration?
+		// TODO: reverify: what about passphrases?
```

Comment:
What's the issue with passphrases? Whitelist should supersede passphrase connections, no?

Reply:
If you have both a whitelist and a passphrase, reverify will kick people who used a passphrase.

Reply:
I think that's fine. Whitelisting is a real security measure, passphrase is a silly one. :P If you're adding whitelisting to the server, then the passphrase has already failed.